bigtable-dataflow-parent/bigtable-beam-import/src/test/java/com/google/cloud/bigtable/beam/hbasesnapshots/EndToEndIT.java

```diff
@@ -345,7 +346,7 @@ public void testHBaseSnapshotImport() throws Exception {
 
     // Validate the counters.
     Map<String, Long> counters = getCountMap(result);
-    Assert.assertEquals(counters.get("ranges_matched"), (Long) 101L);
+    Assert.assertEquals(counters.get("ranges_matched"), (Long) 100L);
```

Comment:
should you add a different snapshot file for snappy?

Reply:
snappy snapshot is generated to have the same data as normal snapshot.